app/containers/MessageBox/UploadModal.js

```diff
@@ -252,4 +252,8 @@ class UploadModal extends Component {
 	}
 }
 
-export default responsive(withTheme(withSplit(UploadModal)));
+const mapStateToProps = state => ({
+	window: state.dimensions.window
+});
+
+export default connect(mapStateToProps)(withTheme(withSplit(UploadModal)));
```

Comment:
You should avoid to add `connect`.
`UploadModal`, for example, has a parent container (`Messagebox`) which declares a `connect`, so it's better to add this `window` props there and pass here as a normal prop.